### PR TITLE
MR-204: automation - add test id for profile of form

### PR DESCRIPTION
**Pull request by VictoriaNegrotto**
added new test id about form on profile of  admin

### CODE REVIEW

File: src/Components/Admin/Admins/Form/index.jsx [hunk 1]
```diff
@@ -97,7 +97,7 @@ const AdminsForm = ({ history }) => {
   return (
     <div>
       <form className={styles.form} onSubmit={handleSubmit(onSubmit, onInvalid)}>
-        <div className={styles.formContainer}>
+        <div className={styles.formContainer} testId="admin-form">
```
Comment (Micaela-Rossi) [inline src/Components/Admin/Admins/Form/index.jsx:100]: following the style we use for name the other tests id this should be admin-profile-form

File: src/Components/Admin/Admins/Form/index.jsx [hunk 2]
```diff
@@ -97,7 +97,7 @@ const AdminsForm = ({ history }) => {
...
           <h1>{text}</h1>
           <fieldset className={styles.fieldset}>
             <Input
```
Comment (Micaela-Rossi) [inline src/Components/Admin/Admins/Form/index.jsx:103]: In the lines 187, 188 and 192 i think you need to add the test id for the buttons like <Button type="confirm" testId="confirm-button"></Button>... i do this, but wait to confirm if it work's in my pr to change it

**Review by Micaela-Rossi — CHANGES_REQUESTED** [review]
SAme as the first file for the second, but i don't know if it's okay because i think vercel doesn't load my changes yet... please wait

File: src/Components/Admin/Admins/Form/index.jsx (hunk 1 above)
Comment (IgnacioCanton) [inline src/Components/Admin/Admins/Form/index.jsx:100]: this should be data-testid="superAdmin-admin-form"

File: src/Components/Admin/Admins/Profile/index.js [hunk 3]
```diff
@@ -84,7 +84,7 @@ const AdminProfile = () => {
   return (
     <div>
       <form className={styles.form} onSubmit={handleSubmit(onSubmit, onInvalid)}>
-        <div className={styles.formContainer}>
+        <div className={styles.formContainer} testId="admin-form">
```
Comment (IgnacioCanton) [inline src/Components/Admin/Admins/Profile/index.js:87]: also this = data-testid="admin-profile-form"

**Review by IgnacioCanton — APPROVED** [review]
looks good to me

**Review by NicolasKler — APPROVED** [review]

**Review by Micaela-Rossi — APPROVED** [review]

**Pull request merged**